crates/iota/src/validator_commands.rs

```diff
@@ -556,7 +556,9 @@ impl IotaValidatorCommand {
                     .await?;
                 let active_validators = match iota_system_state {
```

Comment:
Can this also use `iter_committee_members` function instead of doing a `match`?

Reply:
Same here. Fixed.